scripts/publish.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+echo "Please confirm you have stopped \`make watch\`. (y)es, [N]o: "
+
+read CLEAR;
+
+if [ "_$CLEAR" == "_y" ]; then
+    make prepublish
+    yarn release-tool publish
+    make clean
```

Comment:
Could this be kept in `makefile`? we can `exit 1` when CLEAR is not `y`

Reply:
Addressed in f47e4c6.